TCAT/Views/DatePickerView.swift

```diff
@@ -213,9 +196,6 @@ class DatePickerView: UIView {
     // MARK: - Segment Controls
 
     @objc private func timeTypeSegmentedControlValueChanged(segmentControl: UISegmentedControl) {
-        if timeTypeSegmentedControl.selectedSegmentIndex == arriveByElement.index {
-            leaveNowSegmentedControl.selectedSegmentIndex = UISegmentedControl.noSegment
-        }
     }
```

Comment:
Is this function still being used? It seems like there is no body. If it's not being used just remove it.